cfgov/unprocessed/js/modules/transition/BaseTransition.js

```diff
@@ -0,0 +1,203 @@
+   * Add an event listener to the transition, or call the transition
+   * complete handler immediately if transition not supported.
+   * @param {HTMLNode} elem - Set the HTML element target of this transition.
+   */
+  function setElement( elem ) {
+    // If the element has already been set,
+    // clear the transition classes from the old element.
+    if ( _dom ) {
+      _dom.classList.remove( _classes.BASE_CLASS );
+      flush();
+      animateOn();
+    }
+    _dom = elem;
+    _dom.classList.add( _classes.BASE_CLASS );
+    _transitionEndEvent = _getTransitionEndEvent( _dom );
+  }
+
+  /**
+   * Add a "transition-duration: 0s" utility CSS class.
+   */
+  function animateOn() {
+    _dom.classList.remove( BaseTransition.NO_ANIMATION_CLASS );
+  }
+
+  /**
+   * Remove a "transition-duration: 0s" utility CSS class.
+   */
+  function animateOff() {
+    _dom.classList.add( BaseTransition.NO_ANIMATION_CLASS );
+  }
+
+  /**
+   * @returns {boolean} Whether the transition has a duration or not.
+   */
+  function isAnimated() {
+    return !_dom.classList.contains( BaseTransition.NO_ANIMATION_CLASS );
+  }
+
+  /**
+   * Add an event listener to the transition, or call the transition
+   * complete handler immediately if transition not supported.
+   */
+  function _addEventListener() {
+    // If transition is not supported, call handler directly (IE9/OperaMini).
+    if ( _transitionEndEvent ) {
+      _dom.addEventListener( _transitionEndEvent,
+                                        _transitionCompleteBinded );
+      this.dispatchEvent( BaseTransition.BEGIN_EVENT, { target: this } );
+    } else {
+      this.dispatchEvent( BaseTransition.BEGIN_EVENT, { target: this } );
+      _transitionComplete();
+      this.dispatchEvent( BaseTransition.END_EVENT, { target: this } );
+    }
+  }
+
+  /**
+   * Remove an event listener to the transition.
+   */
+  function _removeEventListener() {
+    _dom.removeEventListener( _transitionEndEvent,
+                                         _transitionCompleteBinded );
+  }
+
+  /**
+   * Handle the end of a transition.
+   */
+  function _transitionComplete() {
+    _removeEventListener();
+    this.dispatchEvent( BaseTransition.END_EVENT, { target: this } );
+  }
+
+  /**
+   * Search for and remove initial BaseTransition classes that have
+   * already been applied to this BaseTransition's target element.
+   * Will only be run once internally, but can be run additional
+   * times externally.
+   */
+  function flush() {
+    for ( var prop in _classes ) {
+      if ( _classes.hasOwnProperty( prop ) &&
+           _classes[prop] !== _classes.BASE_CLASS &&
+           _dom.classList.contains( _classes[prop] ) ) {
+        _dom.classList.remove( _classes[prop] );
+      }
+    }
+  }
+
+  /**
+   * @param {string} clazz - A CSS class.
+   * @returns {boolean} False if the class is already applied,
+   *   otherwise true if the class was applied.
+   */
+  function applyClass( clazz ) {
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/17b34b701b909763b85eed401dbd46e07b0959d1